<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>671</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="3">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">color: rgb(1, 25, 147); font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Laps</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color: rgb(1, 25, 147); font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Speed (mph)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(1, 25, 147); font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Edit Car</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
       <property name="styleSheet">
        <string notr="true">color: rgb(1, 25, 147); font-weight: bold;</string>
       </property>
       <property name="value">
        <double>15.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color: rgb(1, 25, 147); font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Weight (lb)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
       <property name="styleSheet">
        <string notr="true">color: rgb(1, 25, 147); font-weight: bold;</string>
       </property>
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">color: rgb(1, 25, 147); font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Units</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_2">
       <property name="styleSheet">
        <string notr="true">color: white; font-weight: bold;
background-color: rgb(1, 25, 147)</string>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QDoubleSpinBox" name="doubleSpinBox">
       <property name="styleSheet">
        <string notr="true">color: rgb(1, 25, 147); font-weight: bold;</string>
       </property>
       <property name="maximum">
        <double>150.000000000000000</double>
       </property>
       <property name="value">
        <double>73.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white; font-weight: bold;
background-color: rgb(1, 25, 147)</string>
       </property>
       <property name="text">
        <string>View Car Specs</string>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Imperial</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
